### PR TITLE
Updating lecli to fix issue #15 by making all nicknames case insenstive

### PR DESCRIPTION
**Pull request by gearoidof-r7**
Fix for issue #15 . @stopal-r7 @MarkLC-r7

### CODE REVIEW

**Comment by MarkLC-r7** [comment]
+1

File: tests/test_apiutils.py [hunk 1]
```diff
@@ -178,6 +185,13 @@ def test_get_valid_named_group_key():
         assert logkeys == filter(None, str(misc_ex.TEST_LOG_GROUP).splitlines())
 
 
+def test_case_insensitivity_of_named_group_skey():
```
Comment (stopal-r7) [inline tests/test_apiutils.py]: (possible typo) skey?

**Comment by stopal-r7** [comment]
LGTM apart from the possible typo I mentioned above.

**Comment by udangel-r7** [comment]
that should be only one commit (aka squash)

**Comment by gearoidof-r7** [comment]
#squashOnMerge

**Comment by stopal-r7** [comment]
+1

**Pull request merged**